templates/user/settings/appearance.tmpl

```diff
@@ -66,6 +66,25 @@
 			</form>
 		</div>
 
+		<!-- Timestamps -->
+		<h4 class="ui top attached header">
+			{{.locale.Tr "settings.timestamps"}}
```

Comment:
Should we maybe label this box `Interface` to have a generic name where we can later also put things like tab size preference? Or keep it to 1 box per setting? I'm not sure whether multiple boxes per settings will look good, e.g. when we start adding a dropdown for tab size to the box.

Reply:
For reference, here is how it currently looks:

<img width="904" alt="Screenshot 2023-04-28 at 21 15 04" src="https://user-images.githubusercontent.com/115237/235233835-cee63c99-ea93-49a2-acbf-55b6586af7b6.png">

Reply:
I think I like how GitLab does this: Multiple settings and top-aligned label:

<img width="1003" alt="image" src="https://user-images.githubusercontent.com/115237/235234037-c689bdac-f252-4f95-bd96-907908d3c66c.png">

I would recommend we do the same.

Reply:
Maybe we could also just combine the three separate boxes into one. It seems a excessive waste of space to have one box per setting along with a separate save button each. These could be grouped in one box:

<img width="910" alt="image" src="https://user-images.githubusercontent.com/115237/235234953-5fb79b1b-60e0-47cf-806d-128b7ab6ade3.png">

Reply:
I guess if this is too much work, we can defer such an UI change to another PR. Current implementations follows the scheme of the other options, so is consistent with them.

Reply:
Yeah I will open an issue for it after merge.